bug(mdc-experimental): material-components-web needs ^11 but throws error in CLI

#### Reproduction
Steps to reproduce:
1. install and set up mdc-experimental with v11
2. force install material-components-web^11 to fix errors with new mixins used in mdc-experimental theming
3. CLI throws errors looking for material-components-web^9

Is it possible this needs upgrading to material-components-web^11? https://github.com/angular/components/blame/fef9bf52a2de89f62b2f65bba3c7e25a2da8db7e/packages.bzl#L5

```
npm ERR! Found: material-components-web@11.0.0-canary.3201cae47.0
npm ERR! node_modules/material-components-web
npm ERR!   material-components-web@"11.0.0-canary.3201cae47.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer material-components-web@"^9.0.0-canary.419e03572.0" from @angular/material-experimental@11.2.5
npm ERR! node_modules/@angular/material-experimental
npm ERR!   @angular/material-experimental@"^11.2.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```


#### Environment

- Angular: 11.2.4
- CDK/Material: 11.2.3
- Browser(s): chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  mac

--- a/packages.bzl
+++ b/packages.bzl
@@ -2,7 +2,7 @@
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
 ANGULAR_PACKAGE_VERSION = "^12.0.0-0 || ^13.0.0-0"
-MDC_PACKAGE_VERSION = "^9.0.0-canary.419e03572.0"
+MDC_PACKAGE_VERSION = "^11.0.0-canary.0cde52f5a.0"
 TSLIB_PACKAGE_VERSION = "^2.1.0"
 
 # Each placer holder is used to stamp versions during the build process, replacing the key with it's

